### PR TITLE
docs: update tutorials and playground to v19.0.0

### PR DESCRIPTION
**Pull request by devversion**
Updates boilerplate for tutorial and playground to use v19

### CODE REVIEW

**Review by josephperrott — APPROVED** [review]
LGTM

File: adev/src/content/tutorials/deferrable-views/common/package.json [hunk 1]
```diff
@@ -9,20 +9,20 @@
...
     "rxjs": "~7.8.0",
     "tslib": "^2.3.0",
     "zone.js": "~0.15.0"
   },
   "devDependencies": {
-    "@angular/build": "^18.0.1",
-    "@angular/cli": "^18.0.0",
-    "@angular/compiler-cli": "^18.0.0",
-    "typescript": "~5.5.0"
+    "@angular/build": "^19.0.1",
```
Comment (andgdk) [inline adev/src/content/tutorials/deferrable-views/common/package.json]: There are few ocurances of version declaration 19.0.1 for `@angular/build` while most declare 19.0.0

Hope you don't mind mentioning this. Great release :)

Comment (devversion) [reply adev/src/content/tutorials/deferrable-views/common/package.json]: Haha, thank you. Already fixed the fixup commit!

**Comment by devversion** [comment]
Verified locally that this looks good! the dev-preview is taking too long 😄 we can follow-up on fixes

**Review by crisbeto — APPROVED** [review]

**Comment by devversion** [comment]
This PR was merged into the repository by commit 0ce15c6b5e886a8303b1fbbc9b9f6550973c4e3c.

The changes were merged into the following branches: main, 19.0.x

**Pull request closed without merging**